### DISCUSSION TITLE
How to include a large number of icons using relm4-icons?

### DISCUSSION THREAD

**Discussion by gtamas** [Relm4/Relm4#528] [Q&A] [answered]
So relm4-icons generates a feature flag for each icon and only includes those specifically mentioned in Cargo.toml.

However, I wanna allow the users of my app to choose any icon from Gnome icon lib. How can I do this? Do I need to explicitly list them all?

**Comment by AaronErhardt** [comment, marked as answer]
Do I understand this correctly, you want to include all icons in relm4-icons? So far there is no feature flag for this, but it would make sense to add a "all" and per icon-theme feature flags like "gnome-design-icons" to include all. Is this what you are looking for?

**Reply by gtamas** [reply]
Yes exactly. I would like to add all "gnome-design-icons".

**Reply by AaronErhardt** [reply]
I've just added feature flags for entire icon packs and published them in 0.7.0-alpha.2 (only works with Relm4 0.7.0-alpha.1 or Relm4 from git).

**Reply by gtamas** [reply]
Great news! Thx!
